applications/launchpad/gui-react/src/containers/MiningContainer/MiningBoxMerged/index.tsx

```diff
@@ -90,6 +91,7 @@ const MiningBoxMerged = () => {
       nodeState={nodeState}
       containersState={containersState}
       containersToStopOnPause={containersToStopOnPause}
+      helpMessages={MessagesConfig.mergedMiningHelp}
```

Comment:
NIT: would it make sense to switch `mergedMiningHelp` to uppercase? I believe it's more like enum rather than a property/variable. so it would be `MessagesConfig.MergedMiningHelp`

Reply:
Fair point. I took the syntax from other `enum` types in the store and they are all camelCase.

e.g. 

`export enum Settings {
  Mining = 'mining',
  Wallet = 'wallet',
  BaseNode = 'baseNode',
  Docker = 'docker',
  Logs = 'logs',
  Security = 'security',
}`